### PR TITLE
Refactored out PHPExcel library to use PHPSpreadsheet instead

### PR DESCRIPTION
**Pull request by mtshaw3**
Replaced PHPExcel Library with PHPSpreadsheet library

https://github.com/mautic/mautic/issues/8267
https://backlog.acquia.com/browse/MAUT-3201

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -78,8 +78,8 @@
         "friendsofsymfony/oauth-server-bundle": "~1.6.0",
         "willdurand/oauth-server-bundle": "dev-release-0.0.3",
         "jms/serializer-bundle": "~3.4.0",
-        "oneup/uploader-bundle": "~2.0",
-        "phpoffice/phpexcel": "1.8.1",
+        "oneup/uploader-bundle": "~1.3",
```
Comment (escopecz) [inline composer.json]: oneup/uploader-bundle probably shouldn't be downgraded.
```suggestion
        "oneup/uploader-bundle": "~2.0",
```

**Comment by mtshaw3** [comment]
@escopecz  Can you check the conflicts for composer.lock? I was going to update it but i dont know the right changes from the new versions in the composer.json.

**Comment by escopecz** [comment]
Always the higher number. I'll resolve it tomorrow if you won't get to it.

**Comment by mtshaw3** [comment]
> Always the higher number. I'll resolve it tomorrow if you won't get to it.

I understand that, I just wasn't sure about what should happen to the composer.lock file, because the hash has changed.

**Comment by escopecz** [comment]
At this point, I'd remove it and run `composer update` to generate it again. We want the latest libraries for M3 anyway :)

**Comment by escopecz** [comment]
I tested report export to Excel and form submit export to Excel and it works fine. Nice work @mtshaw3! 👍

**Pull request merged**